src/core/InterleavedBufferAttribute.js

```diff
@@ -17,7 +17,7 @@ THREE.InterleavedBufferAttribute.prototype = {
 
 	get length() {
```

Comment:
I take that back because an Attribute is a portion of a Vertex and not the whole thing and this represents the portion